sendinblue.php

```diff
@@ -32,16 +32,19 @@
     include_once (_PS_CLASS_DIR_ . '/../classes/Customer.php');
 }
 
-include_once (dirname(__FILE__) . '/sendinblueResources.php');
 if (version_compare(_PS_VERSION_, '1.4.5', '<')) {
+    require(_PS_CLASS_DIR_. '/../modules/sendinblue/Mailin.php');
```

Comment:
include (dirname(__FILE__) . '/Mailin.php');
this was the good utilisation to catch your file you want to include. Please use dirname(__FILE__).